external/runtime/runtime.depproj

```diff
@@ -42,19 +47,25 @@
     <ItemGroup>
       <HostFxFile Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.Filename)' == '$(HostFxrFileName)'" />
       <DotnetExe Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.Filename)' == 'dotnet'" />
+      <HostSdkFile Include="$(DotNetRoot)sdk\$(DotNetVersion)\**\*" />
```

Comment:
did you consider a directory junction (linux: softlink) instead?  Might be quicker and more efficient than hardlink

Reply:
Symlinks require an elevated process, I would like to avoid such a requirement.

Reply:
@ericstj any concerns with hardlinking?

Reply:
Directory junctions don't require elevation.  Only concern with hardlinking is perf, it's O(n) instead of constant with junctions/symlinks.  If you measure and its reasonable I'm OK with it.

Reply:
500ms in total, I'm ok with that.